src/utils/AudioPlayer.cpp

```diff
@@ -0,0 +1,83 @@
+#include "AudioPlayer.h"
+#include <QDebug>
+
+//#define USE_QSOUNDEFFECT
+#define USE_MEDIAPLAYER
```

Comment:
I have rare problems that ui/application freezes when playing audio using Arch Linux with PulseAudio. 
With QMediaPlayer these hangs occured more often. Using QSoundEffect caused sometimes hangs when in 2 applications the contacts got online simultaniously ( known to each other ).

This issue needs a wider range of testing, preferable also on Windows.